website/project/metadata/osf-standard-2.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Question for understanding: will the bump in version number cause issues for older registration data?

If we alter the structure of the schema file defining questions (such as Eric's schema changes), will we have to alter any pages to deal with:

1. Building a questionnaire for old and new prereg schemas, or
2. Displaying data already populated in the database for old question schemas with a different structure?